index.html

```diff
@@ -70,9 +288,112 @@ <h1 class="name">I’m Ali Hassan. Glad to see you!</h1>
               </li>
             </ul>
           </div>
+          <div class="project-button-container">
+            <a href="#" class="project-button-b">Get my resume</a>
+          </div>
+
+          <div class="about-items-b">
+            <div class="about-knowledge-container">
+              <div class="skill-container" id="active">
+                <div class="skill-title">
+                  <ul>
+                    <li><a href="#" class="link-skill">Langguages</a></li>
+                  </ul>
+                </div>
+                <div class="arrow-down"></div>
```

Comment:
Please try to use the same arrow icons used in the figma design and make sure that they are all facing the directions shown in the figma design

Your design:
<img width="238" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/5c419476-0e71-4d9b-b11f-2876e3df5ef5">

Figma design:
<img width="216" alt="image" src="https://github.com/mhassa204/Portfolio-Project/assets/37116322/b4aa84a2-b7d9-4189-848d-a600dc1bd133">